llm-guard/app/main.py

```
@@ -0,0 +1,105 @@
+    parts = [m.text() for m in req.structured_messages if m.role == "user"]
+    return "\n".join(parts)
+
+
+@app.post("/")
+async def litellm_guardrail(req: LiteLLMRequest):
+    prompt = _extract_prompt(req)
+    if not prompt:
+        return {"action": "NONE"}
+
+    is_safe, score = scanner.scan(prompt)
+    logger.info("litellm scan", extra={"call_id": req.litellm_call_id, "is_safe": is_safe, "score": score})
```

Pylint (4.0.5)

[convention] Line too long (107/100)

(C0301)